src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -1174,92 +1170,97 @@ private static function get_wp_parent_theme(): array {
 			$auto_updates = (array) get_site_option( 'auto_update_themes', array() );
 		}
 
-		$active_theme  = wp_get_theme();
+		$active_theme = wp_get_theme();
 		$parent_theme = $active_theme->parent();
 
-		if ( $parent_theme ) {
-			$parent_theme_version       = $parent_theme->version;
-			$parent_theme_version_debug = $parent_theme_version;
+		if ( ! $parent_theme ) {
```

Comment:
If you want to keep this early return, I'd vote for a stronger condition. `->parent()` will return `false`

So unless you opt to remove it. How about:


`if ( $parent_theme === false ) {`

Reply:
I've replaced the early-abort based on our Slack discussion. here though the flip in logic matches the existing behavior, whereas a strict check for `false` would change it.